cinema/paginations.py

```diff
@@ -0,0 +1,7 @@
+from rest_framework.pagination import PageNumberPagination
+
+
+class OrderPagination(PageNumberPagination):
+    page_size = 1
+    page_size_query_param = "page_size"
+    max_page_size = 1
```

Comment:
The `max_page_size` is set to 1, which might be too restrictive for practical use cases. Consider increasing this value to allow more flexibility in the number of items per page.